### PR TITLE
update version and make package more flexible

### PR DESCRIPTION
**Pull request by mr-tz**

### CODE REVIEW

**Review by MalwareMechanic — CHANGES_REQUESTED** [review]

File: packages/ghidra.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -5,7 +5,7 @@ try {
     $toolName = 'ghidra'
     $category = 'Disassemblers'
     $shimPath = 'bin\ghidra.exe'
-    $versionPath = 'ghidra_10.2.2_PUBLIC'
+    $versionPath = 'ghidra_' + $env:ChocolateyPackageVersion + '_PUBLIC'
```
Comment (mr-tz) [inline packages/ghidra.vm/tools/chocolateyinstall.ps1:8]:
```suggestion
    $versionPath = 'ghidra_' + $env:ChocolateyPackageVersion -replace '\.\d{4}\d{2}\d{2}$' + '_PUBLIC'
```

[the remark this answers is not in the document]
Comment (mr-tz) [reply packages/ghidra.vm/tools/chocolateyinstall.ps1:8]: good catch!

File: packages/ghidra.vm/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -5,7 +5,7 @@ try {
     $toolName = 'ghidra'
     $category = 'Disassemblers'
     $shimPath = 'bin\ghidra.exe'
-    $versionPath = 'ghidra_10.2.2_PUBLIC'
+    $versionPath = 'ghidra_' + $env:ChocolateyPackageVersion -replace '\.\d{4}\d{2}\d{2}$' + '_PUBLIC'
```
Comment (mr-tz) [inline packages/ghidra.vm/tools/chocolateyinstall.ps1:8]:
```suggestion
    # get version as defined in nuspec and remove potential 4th segment `.YYYYMMDD`
    $version = $env:ChocolateyPackageVersion -replace '\.\d{4}\d{2}\d{2}$'
    $versionPath = 'ghidra_' + $version + '_PUBLIC'
```

**Review by MalwareMechanic — APPROVED** [review]
LGTM!

**Pull request merged**